### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by edjrodriguez**
What does this PR do?
* Add guided motion feature as a gif

Where should the reviewer start?
* README

Anything else the reviewer needs to know (e.g. issues, concerns)?
*

### CODE REVIEW

**Review by aspitz1 — APPROVED** [review]
I think this looks great Eddie! Thank you so much!

**Review by tfettig22 — APPROVED** [review]
Thanks for putting this together Eddie!

File: README.md [hunk 1]
```diff
@@ -3,15 +3,17 @@
...
 
-This project utilized asynchronous workflow relying on code reviews within PR requests, daily stand-ups, project board tickets, and Slack among four remote contributors
+The four members of this project collaborated remotely. We used daily stand-ups, screensharing over Zoom, GitHub Pull Requests, Code Reviews, Kanban-style project board tickets, and Slack to remain productive on a tight schedule. With just 11 part-time days to determine our concept, collect design inspiration, wireframe our proposed features, learn new technologies, and build an end-to-end tested frontend app with backend server and database, teamwork and collaboration was vital to achieve our goals.
 
 * Learning Goals:
-    * Explore backend web technologies to develop a functioning database that would allow our application to be accessed on any device with an internet connection.   
-    * Gain further familiarity with React, React Router, and E2E testing with Cypress
+    * Without instruction, learn how to attach a database to our backend in order to persist user-generated data
+    * Become proficient in React, React Router, and E2E testing with Cypress
```
Comment (edjrodriguez) [inline README.md:12]: 💪🏾

File: README.md [hunk 2]
```diff
@@ -3,15 +3,17 @@
...
     * App development using React hooks in place of class components
 
+## External Data APIs
```
Comment (edjrodriguez) [inline README.md:15]: Ooo yeah I totally forgot about this

File: README.md [hunk 3]
```diff
@@ -3,15 +3,17 @@
 [Deployed Link](https://tasterx.herokuapp.com/)
 
 ## Abstract
-TasteRX is an engaging UX born from the idea of curing the world of bad taste.  Do you have good taste in TV shows?  Do you think the world would be a better place if more people watched a particular show?  The TasteRX UI allows users to search for and then 'prescribe' a TV show with a custom message to a fellow human; the created prescription can then easily be shared via URL.   
+Do you have good taste in TV shows?  Do you think the world would be a better place if more people watched a particular show?  TasteRX allows users to search for and then 'prescribe' a TV show to another person. The created prescription can then easily be shared via URL.
```
Comment (edjrodriguez) [inline README.md:6]: *tsk tsk* this was a perfect use case for a semicolon.  My wife even gave me a big kiss this morning when she saw I used it (she's an English teacher so she knows)

**Pull request merged**